ex04.ex

```diff
@@ -39,7 +39,11 @@ defmodule Ex04 do
       [ 1, 2, 3, 4, 5 ]
 
   """
-  def reverse . . . "your code"
+  def reverse([ h | t ]), do: reduce0( t, [h] )
```

Comment:
## -3

I was wanting you to use the standard reduce function defined previously:

~~~ elixir
def reverse(list), do: reduce(list, [], &[ &1 | &2 ])
~~~